build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+set -e
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+SED=${SCRIPT_ROOT}/../../build/sed-in-place
+run_gen() {
+  if [[ ${SKIP_GEN} = true ]]; then
+    echo "SKIP_GEN is set, skipping manifest  generation"
+    exit
+  fi
+  OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+  TMP_FILE="$(mktemp)"
+
+ START_MARKER='# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+ END_MARKER='# THE ABOVE HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+
```

Comment:
Two formatting questions:
- Can we remove some extra blank lines?
- More consistent indentation in this function?

Reply:
> Two formatting questions:
> 
> * Can we remove some extra blank lines?
> * More consistent indentation in this function?

done (both).